packages/cli/src/commands/labs/index.ts

```diff
@@ -5,18 +5,23 @@ import {ux} from '@oclif/core'
 import {sortBy} from 'lodash'
 
 interface Features {
-  currentUser: any,
-  user: any,
-  app?: any,
+  currentUser: Heroku.Account,
+  user: Heroku.AccountFeature,
+  app?: Heroku.AppFeature | null,
 }
 
 function printJSON(features: Heroku.Account | Heroku.AccountFeature | Heroku.AppFeature) {
   ux.log(JSON.stringify(features, null, 2))
 }
 
-function printFeatures(features: any) {
+function printFeatures(features: Heroku.AppFeature[] | Heroku.AccountFeature | null) {
```

Comment:
I'm not seeing where the type `Heroku.AppFeature[]` is being passed in as an arg. The `Features` interface describes only `Heroku.AppFeature | null`. Was this intentional?

Reply:
Initially, it was intentional so that the `for` loop wouldn't error saying that `features` wasn't iterable. However, Ryan had a way better solution here!